src/hooks/screen/useChatScreenHook.ts

```diff
@@ -156,7 +184,8 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           }
         }
 
-        refresh('chat');
+        refreshWithId('chat', selectedChannel?.id);
+        // refresh('chat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refreshWithId` function is used instead of `refresh`. This seems like a targeted refresh based on specific IDs which could potentially improve performance by reducing unnecessary data fetching. However, ensure that this change doesn't introduce any inconsistencies in your application state.



<!-- This is an auto-generated comment by OSS CodeRabbit -->